src/main/java/ch/uzh/ifi/fs22/sel/group15/cozycave/server/controller/UserController.java

```diff
@@ -62,34 +74,51 @@ public UserGetDto createUser(@RequestBody UserPostPutDto userPostPutDto) {
     public UserGetDto findUser(@PathVariable UUID id) {
```

Comment:
Don't we need @AuthenticationPrincipal String authUserId also here, otherwise we would could simply get individual users, no?

Reply:
that's right. but because i didn't implement the authorization part yet, it's not included yet